src/less/pagination/base/pagination.less

```diff
@@ -32,7 +32,13 @@
 }
 
 .pagination__item {
+    background: none;
+    border: none;
+    cursor: pointer;
```

Comment:
Should we leave this to the browser to decide? Or are we trying to mimic a link in all cases?

Reply:
I wanted to make it consistent between `<a>` and `<button>` just incase you were mixing the two.

Reply:
Yeah, in the spirit of the issue this relates to I don't think we'd be mixing them.

Reply:
I think we should force it into a pointer.  The left/right navigation elements will be pointers since they're anchors.  And the expected behavior is some kind of navigation.

Reply:
Sounds good to me.